### ISSUE TITLE
Authentication with Github

### ISSUE THREAD

**Issue by jvm-odoo** [saadpasta/react-blog-github#26]
Add this according to README.md.

Cool things we we'll be able to do with this:
- [ ] Add reactions
- [ ] Add new articles (if user = owner)
- [ ] Edit an article (if user = owner)
- [ ] Add comments
- [ ] Edit comments
- [ ] Add reactions on comments

All of this, directly on the website.

**Comment by saadpasta** [comment]
Yeah, I was also thinking about the same but the main priority is to add reaction and comment through the website we can add all the other to the Todo Section
